tests/installation/release_notes_from_url.pm

```diff
@@ -19,7 +19,13 @@ sub run {
     enter_cmd "zgrep -oh \"Got release notes.*\" /var/log/YaST2/y2log*";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag 'got-releasenotes-URL') {
-        record_soft_failure('bsc#1190711 - Release notes source does NOT match expectations or not found in YaST logs, expected source: URL');
+        my $not_public = script_run('grep -rE "GMC|RC" /README.BETA');
+        unless (get_var('BETA') || $not_public) {
+            die 'Release notes source does NOT match expectations or not found in YaST logs, expected source: URL';
```

Comment:
(neat) this string could be in variable on top to not repeat it.

Reply:
I was wandering if I shoudl leave the same string in both cases. For the second one, I could rephrase to something that will suggest that URL release notes are not ready yet.